### DISCUSSION TITLE
is that poosible to takeover subdomain via mailchimp.com ?

### DISCUSSION THREAD

**Discussion by tayyabqadir877** [EdOverflow/can-i-take-over-xyz#250] [Q&A]
Hi 

a subdomain giving an error,

" We can't find that page 

It looks like you're trying to reach a page that was built by Mailchimp but is no longer active. Want to know more about us? Visit mailchimp.com " 

is that possible to takeover that subdomain? 

Thanks

**Comment by mheranco** [comment]
Hi,

I tested it (had to pay 10$ for the Core Account) and it is NOT vulnerable. You have to set a random CNAME to a mailchimp domain. If this CNAME is not set, it will not connect the domain to your account.

The CNAME looks like this: `us12-6d20b13a-cb1910f2ac1220f96a31cb52e.pages.mailchi.mp`

![image](https://user-images.githubusercontent.com/26740411/169520356-d8467303-7861-4d29-b147-383e33400457.png)

**Reply by reedA201** [reply]
So it works  or not? 
did you takeover  ?

**Reply by Hun33er** [reply]
are you able to take it over

**Comment by med0101z** [comment]
any news guys can we takeover the mailchamp subdomain ?

**Reply by WhiteHhair** [reply]
can you read? already mention above
